### PR TITLE
Remove tablist (Resolves #2349)

### LINKED ISSUE #2349: Usunięcie tablisty
Utrzymanie `tablisty` ostatnio zajmuje coraz więcej zasobów a istnieją pluginy, które pozwalają na zrobienie tego samego co aktualny tab w FG (np. plugin `Tab`) a nawet więcej i łatwiej. Powinniśmy zrobić co najwyżej jakiś default config właśnie np. dla `Tab` aby ludzie wiedzieli jak skonfigurować nowy plugin

### PR DESCRIPTION
**Pull request by Szczurowsky**

### CODE REVIEW

File: plugin/src/main/java/net/dzikoysk/funnyguilds/FunnyGuilds.java [hunk 1]
```diff
@@ -21,47 +19,21 @@
...
-import net.dzikoysk.funnyguilds.listener.EntityDamage;
-import net.dzikoysk.funnyguilds.listener.EntityInteract;
-import net.dzikoysk.funnyguilds.listener.PistonUse;
-import net.dzikoysk.funnyguilds.listener.PlayerChat;
-import net.dzikoysk.funnyguilds.listener.PlayerDeath;
-import net.dzikoysk.funnyguilds.listener.PlayerJoin;
-import net.dzikoysk.funnyguilds.listener.PlayerLogin;
-import net.dzikoysk.funnyguilds.listener.PlayerQuit;
-import net.dzikoysk.funnyguilds.listener.TntProtection;
+import net.dzikoysk.funnyguilds.listener.*;
```
Comment (Rollczi) [inline plugin/src/main/java/net/dzikoysk/funnyguilds/FunnyGuilds.java]: proszę nie używać wildcard

File: plugin/src/main/java/net/dzikoysk/funnyguilds/FunnyGuilds.java [hunk 2]
```diff
@@ -21,47 +19,21 @@
...
-import net.dzikoysk.funnyguilds.listener.region.EntityProtect;
-import net.dzikoysk.funnyguilds.listener.region.GuildHeartProtectionHandler;
-import net.dzikoysk.funnyguilds.listener.region.HangingBreak;
-import net.dzikoysk.funnyguilds.listener.region.HangingPlace;
-import net.dzikoysk.funnyguilds.listener.region.PlayerCommand;
-import net.dzikoysk.funnyguilds.listener.region.PlayerInteract;
-import net.dzikoysk.funnyguilds.listener.region.PlayerMove;
-import net.dzikoysk.funnyguilds.listener.region.PlayerRespawn;
-import net.dzikoysk.funnyguilds.listener.region.PlayerTeleport;
+import net.dzikoysk.funnyguilds.listener.region.*;
```
Comment (Rollczi) [inline plugin/src/main/java/net/dzikoysk/funnyguilds/FunnyGuilds.java]: proszę nie używać wildcard

**Review by Rollczi — CHANGES_REQUESTED** [review]

File: plugin/src/main/java/net/dzikoysk/funnyguilds/config/ConfigurationFactory.java [hunk 3]
```diff
@@ -4,20 +4,10 @@
...
-import net.dzikoysk.funnyguilds.config.serdes.ItemStackTransformer;
-import net.dzikoysk.funnyguilds.config.serdes.MaterialTransformer;
-import net.dzikoysk.funnyguilds.config.serdes.NumberRangeTransformer;
-import net.dzikoysk.funnyguilds.config.serdes.RangeFormattingTransformer;
-import net.dzikoysk.funnyguilds.config.serdes.RawStringTransformer;
-import net.dzikoysk.funnyguilds.config.serdes.SkinTextureSerializer;
-import net.dzikoysk.funnyguilds.config.serdes.VectorSerializer;
-import net.dzikoysk.funnyguilds.config.tablist.TablistConfiguration;
-import net.dzikoysk.funnyguilds.config.tablist.TablistPageSerializer;
+import net.dzikoysk.funnyguilds.config.serdes.*;
```
Comment (Rollczi) [inline plugin/src/main/java/net/dzikoysk/funnyguilds/config/ConfigurationFactory.java]: tu też wildcard

**Review by Rollczi — CHANGES_REQUESTED** [review]

**Review by Rollczi — APPROVED** [review]

File: plugin/src/main/java/net/dzikoysk/funnyguilds/FunnyGuilds.java [hunk 4]
```diff
@@ -2,14 +2,12 @@
 
 import com.google.common.collect.ImmutableSet;
 import eu.okaeri.configs.exception.OkaeriException;
-import java.io.File;
```
Comment (bopke) [inline plugin/src/main/java/net/dzikoysk/funnyguilds/FunnyGuilds.java]: Dear @Szczurowsky,

We wanted to take a moment to express our gratitude for your contribution in deleting another feature from our project, Funnyguilds. Your help in simplifying our codebase is much appreciated, and we're thankful for your hard work and dedication to the project.

However, we have noticed that you've been reorganizing imports, and we wanted to bring to your attention that it's causing a bit of a problem. Our ability to track who added what to files is getting messed up, which could lead to issues down the line.

We wanted to kindly ask that you refrain from reorganizing imports in the future, to help us maintain our codebase more efficiently. We value your contributions to our project, and we want to make sure that we're all working in sync to achieve our goals.

We recognize that your efforts were crucial in making this happen, and we're grateful for your contribution. Your willingness to lend a hand and help us achieve our goals is truly appreciated, and we're lucky to have you as a member of our community.

Once again, thank you for your hard work and dedication to the project. We're excited to see what the future holds.

Best regards,
Bopke

**Review by bopke — CHANGES_REQUESTED** [review]

File: plugin/src/main/java/net/dzikoysk/funnyguilds/FunnyGuilds.java (hunk 4 above)
Comment (Szczurowsky) [reply plugin/src/main/java/net/dzikoysk/funnyguilds/FunnyGuilds.java]: Hi Bopke,

Thanks for mentioning the problem. I deeply understand problematic of mess in imports. However I believe that deleting unused one is also a part of keeping 'em clean and readable.

Best regards,
Szczurowsky

**Review by P3ridot — CHANGES_REQUESTED** [review]
Jeszcze NMS'a usuń z każdej wersji. Jest zbędny

File: plugin/src/main/java/net/dzikoysk/funnyguilds/FunnyGuilds.java [hunk 5]
```diff
@@ -100,21 +96,21 @@
 import panda.std.Result;
 import panda.utilities.ClassUtils;
 
+import java.io.File;
```
Comment (P3ridot) [inline plugin/src/main/java/net/dzikoysk/funnyguilds/FunnyGuilds.java]: Niech ten import wróci na swoje miejsce

File: plugin/src/main/java/net/dzikoysk/funnyguilds/config/ConfigurationFactory.java [hunk 6]
```diff
@@ -4,20 +4,17 @@
...
 import net.dzikoysk.funnyguilds.config.serdes.MaterialTransformer;
-import net.dzikoysk.funnyguilds.config.serdes.NumberRangeTransformer;
 import net.dzikoysk.funnyguilds.config.serdes.RangeFormattingTransformer;
 import net.dzikoysk.funnyguilds.config.serdes.RawStringTransformer;
-import net.dzikoysk.funnyguilds.config.serdes.SkinTextureSerializer;
-import net.dzikoysk.funnyguilds.config.serdes.VectorSerializer;
-import net.dzikoysk.funnyguilds.config.tablist.TablistConfiguration;
-import net.dzikoysk.funnyguilds.config.tablist.TablistPageSerializer;
+
+import java.io.File;
```
Comment (P3ridot) [inline plugin/src/main/java/net/dzikoysk/funnyguilds/config/ConfigurationFactory.java]: Ten też niech wróci na swoje miejsce

File: plugin/src/main/java/net/dzikoysk/funnyguilds/feature/hooks/placeholderapi/PlaceholderAPIHook.java [hunk 7]
```diff
@@ -13,6 +12,8 @@
 import org.jetbrains.annotations.NotNull;
 import panda.std.Option;
 
+import java.util.Locale;
```
Comment (P3ridot) [inline plugin/src/main/java/net/dzikoysk/funnyguilds/feature/hooks/placeholderapi/PlaceholderAPIHook.java]: Ten też niech wróci na swoje miejsce

File: plugin/src/main/java/net/dzikoysk/funnyguilds/feature/placeholders/GeneralPlaceholdersService.java [hunk 8]
```diff
@@ -1,26 +1,24 @@
...
+import net.dzikoysk.funnyguilds.guild.placeholders.GuildPlaceholdersService;
+import net.dzikoysk.funnyguilds.user.User;
+import net.dzikoysk.funnyguilds.user.placeholders.UserPlaceholdersService;
+import panda.std.stream.PandaStream;
 
 import java.time.OffsetDateTime;
 import java.util.Arrays;
 import java.util.List;
 import java.util.Locale;
 import java.util.Set;
```
Comment (P3ridot) [inline plugin/src/main/java/net/dzikoysk/funnyguilds/feature/placeholders/GeneralPlaceholdersService.java:7]: Te też niech wrócą na swoje miejsce

**Review by P3ridot — CHANGES_REQUESTED** [review]

**Review by P3ridot — APPROVED** [review]

**Pull request closed without merging**